pyproject.toml

```diff
@@ -100,3 +100,8 @@ skips = [
 [tool.pytest.ini_options]
 log_format = "%(asctime)s %(levelname)s %(message)s"
 log_date_format = "%Y-%m-%d %H:%M:%S"
+
```

Comment:
Empty commit message, both here and in the previous commit - some reasoning would be nice.